Makefile

```diff
@@ -15,8 +15,32 @@ extract-all:
 	python main.py -i material/siirtokarjalaiset_III.xml -o material/siirtokarjalaiset_III.json
 	python main.py -i material/siirtokarjalaiset_IV.xml -o material/siirtokarjalaiset_IV.json
 
+extract-all-four-processes: ; ${MAKE} -j4 extract-first extract-second extract-third extract-fourth
+
+extract-all-two-processes: ; ${MAKE} -j2 extract-first-pair extract-second-pair
+
+extract-first-pair:
+	${MAKE} extract-first;\
+	${MAKE} extract-second
+
+extract-second-pair:
+	${MAKE} extract-third;\
+	${MAKE} extract-fourth
+
+extract-first:
```

Comment:
Hmm, kannattaisiko katsoa miten välittää tuonne make-taskiin parametrina nuo tiedostonimet? Olisi hieman siistimpää jos tuolla varsinaisessa ajotaskissa vaan sanottaisiin lista näistä tiedostoista ja ajettaisiin joku `extract-file` task neljään kertaan. Ei tarvitsisi erikseen näitä taskeja tänne. 

Mut noh, kelpaa tämä näinkin, voipahan ainakin makella helposti extractata minkä tahansa noista kirjoista haluttaessa.